skyportal/handlers/api/invitations.py

```diff
@@ -166,6 +173,13 @@ def post(self):
                 "Invalid value provided for `canSave` parameter: "
                 "all elements must be booleans"
             )
+        user_expiration_date = data.get("userExpirationDate")
+        if user_expiration_date is not None:
+            try:
+                user_expiration_date = arrow.get(user_expiration_date).datetime
```

Comment:
Does arrow know to use UTC by default?

Reply:
Yup, looks like it does:
![image](https://user-images.githubusercontent.com/17696889/124020673-93b11d00-d9b8-11eb-9d03-56934b6742f1.png)